Representable/representable
# Allowlist flow

**Changes:**

- add private field to Drive
- add private field to CommunityEntry
- update AllowList to have a Drive field
- make CommunityEntry privacy match Drive privacy
- redirect from Drive entry to page to generic entry page if user is not on private drive's allowlist
- change frontend to allowlist and allowlist flow:
  - move allowlist away from org home
  - call it an invite list
  - have it only on private drives
  - add a "manage invite list" page
  - add a list of members on Manage page
  - allow csv uploads to add to allowlist
  - add an "add" button to add a single email address to the allowlist
  - add a remove button to allow org admins to remove members from the invite list
- hide private communities from everyone who is not an admin of the org to which they belong
- add "Private" tag to drive home

**To test:**
- migrate
- runserver 
- create a drive, and update private to True in postico or however you prefer to access your database
- see that the drive page for that drive includes a "private" tag
- look at "Manage Invite List"
- upload a csv of emails and see that they show up under "Invitees" 
- add an email using the add textbox-and-button and see that it shows up under "Invitees"
- click on the trashcan and remove a member and see that that email no longer is under "Invitees"
- try entering the same email twice and see that it doesn't appear a second time
- submit a community at /entry/drive/<drive_slug> 
- view drive/org map as org admin, see the private community
- view drive/org map as not org admin, check that there are no private communities 

**To do:**
- send an email with link to Invitees on being added (I need help with the flow of this especially with verification)
